### ISSUE TITLE
False negative for Style/MultipleComparison when non-equality comparison present

### ISSUE THREAD

**Issue by asherkach** [rubocop-hq/rubocop#5466]
The cop `Style/MultipleComparison` fails to trigger when several equality comparisons are followed by a non-equality comparison.

--------

## Expected behavior

A violation is reported for
```
def foo(x)
  if x == 1 || x == 2 || x == 3 || x.nil?
    1
  end
end
```
as this can safely be "fixed" to
```
def foo(x)
  if ([1, 2, 3].include? x) || x.nil?
    1
  end
end
```

## Actual behavior

No violation is reported.

## RuboCop version
```
$ rubocop -V
0.52.1 (using Parser 2.4.0.2, running on ruby 2.2.3 x86_64-darwin16)
```

--------

See the discussion [here](https://github.com/code-dot-org/code-dot-org/pull/20006) for context.
